modules/liburing/2.6/source.json

```
@@ -0,0 +1,10 @@
+{
+    "url": "https://github.com/axboe/liburing/archive/refs/tags/liburing-2.6.tar.gz",
+    "integrity": "sha256-aC8Gcz5ttkAsH5BMu+ErlJQqSe/8hyyeAds9exgJF8w=",
+    "strip_prefix": "liburing-liburing-2.6",
+    "patches": {
+        "add_build_file.patch": "sha256-bSFDyIEyJYYP0sXNNePKjbnLKGMveouIm6pZ6fRY9Yc=",
```

Could you try this for me?

```suggestion
        "../../2.5/patches/add_build_file.patch": "sha256-bSFDyIEyJYYP0sXNNePKjbnLKGMveouIm6pZ6fRY9Yc=",
```
and remove the new patch file. 

If that works, it would make it much easier to tell that the patch didn't change.